docs/tutorials/intro_tutorial.rst

```diff
@@ -384,213 +348,208 @@ those on the left edge, and the top to the bottom. This prevents some
 a grid and make them walk around at random. Instead of giving their unit
-of money to any random agent, they'll give it to an agent on the same
+of money to any random agent, they’ll give it to an agent on the same
 cell.
 
 Mesa has two main types of grids: ``SingleGrid`` and ``MultiGrid``.
 ``SingleGrid`` enforces at most one agent per cell; ``MultiGrid`` allows
 multiple agents to be in the same cell. Since we want agents to be able
 to share a cell, we use ``MultiGrid``.
 
-.. code:: python
+.. code:: ipython3
 
     from mesa.space import MultiGrid
 
 We instantiate a grid with width and height parameters, and a boolean as
-to whether the grid is toroidal. Let's make width and height model
+to whether the grid is toroidal. Let’s make width and height model
 parameters, in addition to the number of agents, and have the grid
-always be toroidal. We can place agents on a grid with the grid's
+always be toroidal. We can place agents on a grid with the grid’s
 ``place_agent`` method, which takes an agent and an (x, y) tuple of the
 coordinates to place the agent.
 
-.. code:: python
+.. code:: ipython3
 
     class MoneyModel(Model):
         """A model with some number of agents."""
         def __init__(self, N, width, height):
             self.num_agents = N
             self.grid = MultiGrid(width, height, True)
             self.schedule = RandomActivation(self)
-
+            
             # Create agents
             for i in range(self.num_agents):
                 a = MoneyAgent(i, self)
                 self.schedule.add(a)
-
+                
                 # Add the agent to a random grid cell
                 x = self.random.randrange(self.grid.width)
                 y = self.random.randrange(self.grid.height)
                 self.grid.place_agent(a, (x, y))
 
-Under the hood, each agent's position is stored in two ways: the agent
+Under the hood, each agent’s position is stored in two ways: the agent
 is contained in the grid in the cell it is currently in, and the agent
 has a ``pos`` variable with an (x, y) coordinate tuple. The
 ``place_agent`` method adds the coordinate to the agent automatically.
 
-Now we need to add to the agents' behaviors, letting them move around
+Now we need to add to the agents’ behaviors, letting them move around
 and only give money to other agents in the same cell.
 
-First let's handle movement, and have the agents move to a neighboring
-cell. The grid object provides a ``move_agent`` method, which like you'd
+First let’s handle movement, and have the agents move to a neighboring
+cell. The grid object provides a ``move_agent`` method, which like you’d
 imagine, moves an agent to a given cell. That still leaves us to get the
 possible neighboring cells to move to. There are a couple ways to do
 this. One is to use the current coordinates, and loop over all
 coordinates +/- 1 away from it. For example:
 
 .. code:: python
 
-    neighbors = []
```

Comment:
Here and below: Why change the indentation from 4 spaces to 3 spaces? Any special meaning to that?

Reply:
No special meaning, most likely inadvertent

Reply:
Since it is consistent I don't think it is an issue, but if you don't mind could you change it back?